src/server/services/hli/hli-status-line-items.ts

```diff
@@ -25,7 +24,7 @@ export const hliStatusLineItemsConfig: ZorgnedStatusLineItemsConfig<
   ZorgnedHLIRegeling | ZorgnedAanvraagWithRelatedPersonsTransformed
 >[] = [
   {
-    productIdentificatie: [AV_UPCC, AV_UPCZIL, AV_PCVC, AV_PCVZIL],
+    productIdentificatie: [AV_UPCC, AV_PCVC, ...verzilveringCodes],
```

Comment:
Ik zie toch de codes expliciet vermeld hier. We gebruiken deze file ook om te kijken welke codes er zijn geïmplementeerd.

Reply:
Kan niet ivm feature toggles!